### DISCUSSION TITLE
There's some way to create a element on Leptos with document().create_element() and add it at the view?

### DISCUSSION THREAD

**Discussion by leorsousa05** [leptos-rs/leptos#1551] [Q&A] [answered]
I wanted to create a element on Leptos and add it into the view! I wanted to do it because the leptos don't have a html to RSX parser, so it's impossible to get some text from a API and just transform it to HTML. Have some other way to do it? Add a string html to leptos view?

**Comment by gbj** [comment, marked as answer]
Not exactly what you're asking, but by far the best way to get some HTML from an API and insert it into the DOM:
```rust
let html = "<span>Here's some HTML</span>";
view! { cx, <div inner_html=html/> }
```
